Misc-Tools.lua

```diff
@@ -117,66 +161,102 @@ local function OnCreateMove(pCmd)
 
     -- Random Fakelag
     if mRandLag:GetValue() == true then
-        if pLocal:IsAlive() and pCmd.command_number % 20 == 0 then
-            local randValue = math.random(120, 315)
+        flTimer = flTimer +1
+        if (flTimer >= mRandLagValue:GetValue()) then
+            flTimer = 0
+            local randValue = math.random(mRandLagMin:GetValue(), mRandLagMax:GetValue())
             gui.SetValue("fake lag value", randValue)
         end
     end
 
+    -- Random Ping
+    if mRandPing:GetValue() == true then
+        prTimer = prTimer +1
+        if (prTimer >= mRandPingValue:GetValue() * 66) then
```

Comment:
Why not use `pCmd.command_number % (mRandPingValue:GetValue() * 66) == 0`?

Reply:
See https://github.com/lnx00/Lmaobox-LUA/pull/4#discussion_r894529633